geode-core/src/main/java/org/apache/geode/internal/cache/xmlcache/RegionAttributesCreation.java

```diff
@@ -1576,7 +1576,6 @@ public void setPartitionAttributes(PartitionAttributes partitionAttr) {
       }
 
       setHasPartitionAttributes(true);
-      ((PartitionAttributesImpl) partitionAttributes).setOffHeap(offHeap);
```

Comment:
Don't know much about this code; but looking at the new test added; it looks like the offHeap property is getting set in some other place.

Reply:
@agingade : yes, `offHeap` can't be effectively set until the member is up and running and we have a distributed system created. I've wrongly introduced this line here while working on [GEODE-7162](https://issues.apache.org/jira/browse/GEODE-7162).